### PR TITLE
[new locale] tet: Tetun Dili (East Timor) locale

### PR DESCRIPTION
**Pull request by joshbrooks**
Credit to @marobo for initial release

### CODE REVIEW

File: locale/tdt.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+//! moment.js locale configuration
+//! locale : Tetun Dili (East Timor) [tdt]
```
Comment (ichernev) [inline locale/tdt.js]: you need to add an author line here (check the other locales for example). A good place to credit @marobo or anybody else who helped.

**Comment by mj1856** [comment]
A couple of points:

1. The [ISO 639-2 code](https://www.loc.gov/standards/iso639-2/php/code_list.php) for the Tetum language is `tet`, not `tdt`.

2. You are creating the file in the `/locale`, folder, which is the output folder for the build process.  Please create the file in `/src/locale`, and a corresponding unit test in `/src/test/locale`.  Refer to [the contributing guidelines](https://github.com/moment/moment/blob/develop/CONTRIBUTING.md).

**Comment by joshbrooks** [comment]
We use 'tdt' for some of our internal work ('Tetum dili') but tet is a better option as it's a recognised -2 code (tdt being a -3). Filenames have been changed to suit.
Files have been moved to source folder as requested, and tests are all passing. Thanks for the help @mj1856

**Comment by ichernev** [comment]
@joshbrooks thank you. This will go out in next release.
I just need you to sign the CLA.

**Comment by ichernev** [comment]
@joshbrooks  can you please sign the CLA. I can not proceed otherwise.

**Comment by joshbrooks** [comment]
@ichernev CLA has been signed, Apologies for the delay and thank you.

**Comment by ichernev** [comment]
Merged in 270d7f2b1e28fd3a36a9c0d154fffc1af5885f78

**Pull request closed without merging**